### DISCUSSION TITLE
TuYa CX-0726 is showing as Not supported (EndDevice)

### DISCUSSION THREAD

**Discussion by spysky** [Koenkk/zigbee2mqtt#14943] [Devices]
I'm using the latest build, and I'm unable to add TuYa CX-0726.

According to documentation, this device was added back in 1.25.2 , but when I add it I get this in the log
"Zigbee2MQTT:info  2022-11-12 12:58:09: 0xa4c1381a0e086acd (0xa4c1381a0e086acd): Not supported (EndDevice)"

Can anyone help with this?

Don't know if it helps but, I'm running zigbee2mqtt on Docker linux/arm64

**Comment by Koenkk** [comment]
Can you provide the data/database.db entry of your device?

**Comment by spysky** [comment]
Sure
[database.zip](https://github.com/Koenkk/zigbee2mqtt/files/9995806/database.zip)

**Reply by Koenkk** [reply]
The device did not pair successfully, force remove it via the z2m frontend, restart z2m and try pairing it again close to the coordinator.

**Reply by spysky** [reply]
Published this message "{"id":"0xa4c1381a0e086acd","force":true}" on topic "zigbee2mqtt/bridge/config/force_remove"

This was the log output
"Zigbee2MQTT:debug 2022-11-13 10:36:28: Received MQTT message on 'zigbee2mqtt/bridge/config/force_remove' with data '{"id":"0xa4c1381a0e086acd","force":true}'
Zigbee2MQTT:error 2022-11-13 10:36:28: Cannot force remove, device '{"id":"0xa4c1381a0e086acd","force":true}' does not exist"

Did I miss anything?

**Reply by Koenkk** [reply]
Then device probably already left, try repairing it now.
